src/test/rustdoc-gui/search-tab-selection-if-current-is-empty.goml

```diff
@@ -0,0 +1,21 @@
+goto: file://|DOC_PATH|/index.html
+write: (".search-input", "Foo")
+// Waiting for the search results to appear...
+wait-for: "#titles"
```

Comment:
Does this have a timeout? Or will it hang forever if it doesn't appear?

Reply:
30 seconds by default. You need to use the `timeout` command with `0` to set to "infinite" wait.